substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/StringEntry.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2020, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2020 Red Hat, Inc. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package com.oracle.objectfile.elf.dwarf;
+
+/**
+ * Used to retain a unique (up to equals) copy of a
+ * String. Also flag swhether the String needs to be
+ * located in the debug_string section and, if so,
+ * tracks the offset at which it gets written.
+ */
+public class StringEntry {
+    private String string;
+    private int offset;
+    private boolean addToStrSection;
+
+    StringEntry(String string) {
+        this.string = string;
+        this.offset = -1;
+    }
+
+    public String getString() {
+        return string;
+    }
+
+    public int getOffset() {
+        /*
+         * offset must be set before this can be fetched
```

Comment:
I still see many of these comments starting with lowercase letters

Reply:
Yes. I cleaned up a lot of them before fixing the other issues but I have not yet done a second pass over them. I will do that as part of the next cleanup that handles all this feedback.